contrib/docker/docker-compose.yml

```diff
@@ -1,4 +1,8 @@
 # docker-compose build && docker-compose up
+# If you experience problems with the CKAN container not being
+# able to connect to the DB, then most likely the DB has not
+# started up quickly enough. Just do "docker-compose up ckan"
+# again to retry
```

Comment:
but probably this one can be used in ckan-entrypoint.sh, rather simple, but we need to install netcat package as well
https://github.com/GSA/catalog-app/blob/master/docker/webserver/entrypoint.sh#L12-L14